templates/rhpam70-authoring.yaml

```diff
@@ -198,6 +198,8 @@ parameters:
   description: Password to access the Maven repository.
   name: MAVEN_REPO_PASSWORD
   from: "[a-zA-Z]{6}[0-9]{1}!"
+  generate: expression
+  required: true
```

Comment:
This is not needed, if it is not provided it will be generated and that's why you're adding the `generate: expression` tag

Reply:
this should be required: false